### PR TITLE
Create 2015-03-18-Wifi-pairing.md

### PR DESCRIPTION
**Pull request by DrJukka**
---

layout: post
## title: Wifi pairing

I had [previously](http://www.drjukka.com/blog/wordpress/?p=39) suggested that one could teach a WiFi Direct device about "known" groups creating a "pairing" experience similar to Bluetooth and getting rid of multiple wi-fi acceptance dialogs.

Finally I got to run tests on the idea, and results are [here](http://www.drjukka.com/blog/wordpress/?p=46).

### CODE REVIEW

File: _posts/2015-03-18-Wifi-pairing.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Wifi pairing
+---
+Earlier I suggested that the Known groups for Wifi Direct could be teached for the device.
+Thus making the Wifi direct "pairable" same way as Bluetooth.
```
Comment (yaronyg) [inline _posts/2015-03-18-Wifi-pairing.md]: I found this sentence hard to follow. I think what you mean is something like "Wifi Direct on Android requires user confirmation to establish a connection. But as explained [here](http://www.drjukka.com/blog/wordpress/?p=46) we can treat this like Bluetooth pairing. Once two devices are 'paired' they can communicate over Wifi Direct in the background or the foreground without any further user confirmations! This opens up Wifi Direct's longer range and higher bandwidth as an alternative to unauthenticated Bluetooth connections."

Comment (DrJukka) [inline _posts/2015-03-18-Wifi-pairing.md]: Yep, did that on hurry, thus was not thinking how it sounds like. Yours sounds lot better

Comment (yaronyg) [inline _posts/2015-03-18-Wifi-pairing.md]: Can you please update and resubmit the PR?
